rados/snapshot.go

```diff
@@ -70,3 +75,77 @@ func (snapshot *Snapshot) Remove() error {
 	ret := C.rados_ioctx_snap_remove(snapshot.ioctx.ioctx, cSnapName)
 	return getError(ret)
 }
+
+// SnapID represents the ID of a rados snapshot.
+type SnapID C.rados_snap_t
+
+// LookupSnapshot gets the ID of a pool snapshot.
+//
+// Implements:
+//  int rados_ioctx_snap_lookup(rados_ioctx_t io, const char *name, rados_snap_t *id)
+func (snapshot *Snapshot) LookupSnapshot() (SnapID, error) {
```

Comment:
Maybe name this function just `Lookup()` as this is a method of the `Snapshot` type.

Reply:
Fixed